docs/configuration/configuration.md

```diff
@@ -387,8 +387,8 @@ See below for the configuration options for EC2 discovery:
 ```yaml
 # The information to access the EC2 API.
 
-# The AWS Region.
-region: <string>
+# The AWS Region. If blank, will try to fetch from metadata service.
```

Comment:
This is not a guaranteed behaviour of Prometheus, merely something the library we are using does itself.

Reply:
I disagree https://github.com/serathius/prometheus/blob/master/discovery/ec2/ec2.go#L99

Reply:
It's still not something we guarantee, configuration is meant to come entirely from the config file. That code really shouldn't be there.